JonathanCoco/ninjagold/apps/ninjagoldapp/templates/ninjagoldapp/index.html

```diff
@@ -0,0 +1,63 @@
+<html>
+    <head>
+        <title>Ninja Money</title>
+        {% load staticfiles %}
+        <link rel="stylesheet" href="{% static 'ninjagoldapp/styles.css' %}">
+    </head>
+    <body>
+        <div id="container">
+            <p>Your Gold... {{request.session.your_gold}}</p>
+            <form action="/reset" method="post">
+                {% csrf_token %}
+                <input type="submit" value="reset"/>
+            </form>
+
+            <div class="stores">
+                <form action="/process_money/farm" method="post">
+                    {% csrf_token %}
+                    <p>Farm</p>
+                    <p>(earns 10-20 golds)</p>
+                    <!-- <input type="hidden" name="building" value="farm"/> -->
+                    <input type="submit" value="Find Gold!"/>
+                </form>
+            </div>
+
+            <div class="stores">
+                <form action="/process_money/cave" method="post">
+                    {% csrf_token %}
+                    <p>Cave</p>
+                    <p>(earns 5-10 golds)</p>
+                    <!-- <input type="hidden" name="building" value="cave"/> -->
+                    <input type="submit" value="Find Gold!"/>
+                </form>
+            </div>
+
+            <div class="stores">
+                <form action="/process_money/house" method="post">
+                    {% csrf_token %}
+                    <p>House</p>
+                    <p>(earns 2-5 golds)</p>
+                    <!-- <input type="hidden" name="building" value="house"/> -->
+                    <input type="submit" value="Find Gold!"/>
+                </form>
+            </div>
+
+            <div class="stores">
+                <form action="/process_money/casino" method="post">
+                    {% csrf_token %}
+                    <p>Casino</p>
+                    <p>(earns/takes 0-50 golds)</p>
+                    <!-- <input type="hidden" name="building" value="casino"/> -->
+                    <input type="submit" value="Find Gold!"/>
+                </form>
+            </div>
+
+            <div id="activities">
+                <select id="ActivityList" size="10">
+                    {% for activity in request.session.your_activities %}
+                        <option value="1" class={{activity.color}}>{{activity.activity}}</option>
+                    {%endfor%}
+                </select>
+            </div>
+    </body>
+</html>
```

Comment:
Nice work using the url to pass what 'building' you're using.  But since you're not actually sending any data that way, you can get rid of all the forms and just send GET requests using an anchor link tag